### PR TITLE
Ft/add maximum draggable value

### PR DESCRIPTION
**Pull request by phamhoaivu911**
# Summary
Add maximumDraggableValue prop.

### CODE REVIEW

File: src/Slider.js [hunk 1]
```diff
@@ -451,17 +457,24 @@ export default class Slider extends PureComponent {
...
   _setCurrentValueAnimated = (value: number) => {
+    const maximumDraggableValue = this.props.maximumDraggableValue;
     const animationType = this.props.animationType;
     const animationConfig = Object.assign(
       {},
       DEFAULT_ANIMATION_CONFIGS[animationType],
       this.props.animationConfig,
       {
-        toValue: value,
+        toValue: value <= maximumDraggableValue ? value: maximumDraggableValue,
```
Comment (ehhuythai) [inline src/Slider.js]: space before the `:`?

File: src/Slider.js [hunk 2]
```diff
@@ -451,17 +457,24 @@ export default class Slider extends PureComponent {
   _getCurrentValue = () => this.state.value.__getValue();
 
   _setCurrentValue = (value: number) => {
-    this.state.value.setValue(value);
+    const maximumDraggableValue = this.props.maximumDraggableValue;
+
+    if(value <= maximumDraggableValue) {
```
Comment (ehhuythai) [inline src/Slider.js]: is it better to have a function `toValue(value)` and share it with the below?

File: src/Slider.js (hunk 1 above)
Comment (phamhoaivu911) [reply src/Slider.js]: 🤦‍♂️

File: src/Slider.js (hunk 2 above)
Comment (toan2406) [reply src/Slider.js]: space after `if`? :trollface:

**Pull request merged**